hubspot3/contacts.py

```diff
@@ -5,7 +5,7 @@
 from typing import Union
 from hubspot3.crm_associations import CRMAssociationsClient
 from hubspot3.base import BaseClient
-from hubspot3.utils import prettify, get_log
+from hubspot3.utils import prettify, get_log, clean_result
```

Comment:
```suggestion
from hubspot3.utils import clean_result, get_log, prettify
```
Colocando em ordem alfabética aqui (o flake8 deve dar um warning).